source/fundamentals/transactions.txt

```diff
@@ -256,9 +251,26 @@ shows how you can perform the following:
    :start-after: start placeOrder
    :end-before: end placeOrder
 
-Note that you must pass the session object to each CRUD operation that
+You must pass the session object to each CRUD operation that
 you want to run on that session.
 
+.. important:: Use a Session with the Client that Started It
```

Comment:
I: "that" is a pronoun here and [should be capitalized](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#guidelines-for-headline-style-capitalization)

```suggestion
.. important:: Use a Session with the Client That Started It
```